src/inject/dynamic-theme/index.ts

```diff
@@ -44,13 +43,22 @@ function createOrUpdateStyle(className: string, root: ParentNode = document.head
     return element;
 }
 
-function createOrUpdateScript(className: string, root: ParentNode = document.head || document) {
+/**
+ * Note: src must be included in web_accessible_resources manifest key.
+ */
+function createOrUpdateScript(className: string, src: string, args: any = undefined, root: ParentNode = document.head || document) {
     let element: HTMLScriptElement = root.querySelector(`.${className}`);
```

Comment:
I'm not sure why we attempt to preserve the same `<script>` element if we end up removing it anyway... Can we just not do this?

Reply:
I fixed this already.